docs/source/getting_started/getting_started.md

```diff
@@ -423,7 +390,269 @@ curl -X POST "$URL/users/u1234/personalized_documents" \
     }'
 ```
 
-# Candidates
+Filter APIs can also work with rfc3339 formatted strings if the field is of type `date`:
+
+```{code-block} bash
+:caption: /users/{user_id}/personalized_documents
+
+curl -X POST "$URL/users/u1234/personalized_documents" \
+    --header "authorizationToken: $FRONTOFFICE_TOKEN" \
+    --header "Content-Type: application/json" \
+    --data '{
+        "filter": {
+            "publication_date": {
+                "$gte": "2022-12-30T11:30:32Z"
+            }
+        },
+        "include_properties": true
+    }'
+```
+
+# Documents with multiple Snippets
+
+Documents can have multiple snippets, and at the core our search and recommendation system is based on the searching for and recommending snippets not documents.
+
+When using documents with multiple snippets you need to be aware about a few things:
+
+- there is a snippet id for each specific snippet a document has
+- we search for snippets not documents, so results can contains multiple (different) snippets of the same document
+- many APIs which accepts either a document id for referring to a document as a whole or a snippet id to refer to a specific snippet
+    - some APIs currently fall back to using the first snippet if only document id is used, but this is intended to change in the future
+
+We made sure that all search APIs in which you can refer to a specific snippet also work with just the document id even if the document has multiple snippets. _But_ the results might not be quite as expected as there can be a huge difference in the search results.
```

Comment:
```suggestion
```suggestion
- APIs accept either a document id for referring to a document as a whole or a snippet id to refer to a specific snippet. Currently, the document id will fall back on the first snippet of the document; this means that the documents in the results could be quite different from the expected one.
```

I feel that the two sentences were partially a repetition of each other but also in contradiction since one says that many (relevant) API get both and the other that all API can work with both

Reply:
> Currently, the document id will fall back on the first snippet of the document;

This is not correct. Some do, others already do something better, through not necessary flashed out.

>  other that all API can work with both

not all which can be called with document id (e.g. candidates API) can be called with snippet id

but all which work with snippet id can be called with document id 

of the ones which can be called with snippet id most have suboptimal results when called with document id of documents with multiple snippets but with varying degrees of suboptimality

some of the suboptimal results are just imbalanced, e.g. when computing CoI tof documents with multiple snippets as a whole it biases in favor of documents with more snippets

some other do fall back to using one snippet of the document, mainly (only?) the semantic search 



and some of the ones which work with both have suboptimal implementations in various ways when called with a document id for cases where the document referred to has multiple snippets, e.g. some of the some do fall back to using the first snippet

Reply:
I rephrased it I hope it's now more clear.